src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static final Fruit CHERRY = new Fruit("cherry");
+    private static final Integer CHERRY_QUANTITY = 150;
+    private static final Fruit BANANA = new Fruit("banana");
+    private static final Integer BANANA_QUANTITY = 50;
+    private static final Fruit BLUEBERRY = new Fruit("blueberry");
+    private static final Integer BLUEBERRY_QUANTITY = 23;
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+        fruitDao.update(CHERRY, CHERRY_QUANTITY);
+        fruitDao.update(BANANA, BANANA_QUANTITY);
+        fruitDao.update(BLUEBERRY, BLUEBERRY_QUANTITY);
+    }
+
+    @Test
+    public void updateTest_Ok() {
+        Map<Fruit, Integer> expected = new HashMap<>();
+        expected.put(new Fruit("cherry"), 150);
+        expected.put(new Fruit("banana"), 50);
+        expected.put(new Fruit("blueberry"), 23);
+
+        Map<Fruit, Integer> actual = Storage.getFruits();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getTest_Ok() {
```

Comment:
I am not sure but the name of this test a bit confusing.

Reply:
done